state/txindex/kv/kv_test.go

```diff
@@ -221,60 +221,6 @@ func TestTxSearchOneTxWithMultipleSameTagsButDifferentValues(t *testing.T) {
 	assert.Equal(t, []*types.TxResult{txResult}, results)
 }
 
-func TestTxSearchMultipleTxs(t *testing.T) {
```

Comment:
why remove??

Reply:
Because we have removed https://github.com/tendermint/tendermint/pull/4253#discussion_r358703634

Reply:
RIght. But we still want to test "indexed fourth (to test we don't include txs with similar events)" use-case. See https://github.com/tendermint/tendermint/issues/2908

Reply:
So should I take back https://github.com/tendermint/tendermint/pull/4253#discussion_r358703634?